deploy/config-common.ts

```diff
@@ -60,4 +62,17 @@ export async function assertWardenHandlerConfigValidity(
 
   const axelarGateway = IAxelarGateway__factory.connect(config.axelarGateway, provider);
   await axelarGateway.tokenAddresses('WETH'); // throws an error if address has no right method hash
+
+  const isLowerCase = config.wardenContractAddress === config.wardenContractAddress.toLowerCase();
+  const isUpperCase = config.wardenContractAddress === config.wardenContractAddress.toUpperCase();
+
+  if (!isLowerCase && !isUpperCase) {
+    throw new Error(
+      `Invalid warden contract address ${config.wardenContractAddress}: must be either upper or lower case`
+    );
+  }
+  const decoded = bech32.decode(config.wardenContractAddress);
+  if (decoded.prefix != 'warden' || decoded.words.length != 32) {
```

Comment:
Will `bech32.decode(config.wardenContractAddress.toUpperCase())` return prefix in lower case?